config/cms.php

```diff
@@ -93,6 +93,19 @@
 
     'backendSkin' => 'Backend\Skins\Standard',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Automatically run migrations on login
+    |--------------------------------------------------------------------------
+    |
+    | If value is true, UpdateMananger will be run on logging in to the backend.
+    | It's recommended to set this value to 'false' in production enviroments
+    | because it clears the cache every time a user logs in to the backend.
+    |
+    */
+
+    'runMigrationsOnLogin' => true,
```

Comment:
@bennothommo I wonder if we should have this set to null by default to represent pulling its value from the value of app.debug (debug enabled this enabled, debug disabled this disabled). Thoughts? Are users still able to have migrations run when installing new plugins or plugin updates through the backend when debug (i.e. this) is disabled in that case?

Reply:
@LukeTowers Users can still use the *Check for updates* button in the *Plugins & Updates* area of the Settings to run migrations, so I agree this would be a good change to have. Migrations shouldn't automatically run on production.

Reply:
@bennothommo @Samuell1 could you make the change?